File: TimeToPitch/Controller.m
#import "Controller.h"
#import "ConvController.h"

@implementation Controller

- (IBAction)Convert:(id)sender
{
	float CurTempo, DesTempo, DetCents;
	CurTempo=[fieldCurrTempo floatValue];
	DesTempo=[fieldDesTempo floatValue];
	DetCents=[bridge Calculate:CurTempo:DesTempo];
	[fieldDetCents setFloatValue:DetCents];
	[fieldCurrTempo selectText:self];
	[fieldDesTempo selectText:self];
}

@end

File: TimeToPitchJava/Controller.java
/* Controller */

import com.apple.cocoa.foundation.*;
import com.apple.cocoa.application.*;

public class Controller extends NSObject {

    public ControllerMath bridge; /* IBOutlet */
    public NSTextField fieldCurTempo; /* IBOutlet */
    public NSTextField fieldDesTempo; /* IBOutlet */
    public NSTextField fieldDetCents; /* IBOutlet */

    public void Convert(Object sender) { /* IBAction */
		float CurSampleTempo, DesSampleTempo;
		double DetCents;
		CurSampleTempo=fieldCurTempo.floatValue();
		DesSampleTempo=fieldDesTempo.floatValue();
		DetCents=bridge.calcola(CurSampleTempo,DesSampleTempo);
		fieldDetCents.setDoubleValue(DetCents);
		fieldCurTempo.selectText(this);
		fieldDesTempo.selectText(this);
    }

}
